feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt

```diff
@@ -31,30 +31,26 @@ class AnnouncementStoreTest {
         val observer = mockk<(LoadingState?) -> Unit>(relaxed = true)
 
         announcementStore.loadingState.changedForever(observer)
+        coVerify { observer(LoadingState.INITIALIZED) }
```

Comment:
I'm curious so let me ask one question, is this `coVerify` needed? Didn't `verify` work then?

Reply:
I will check that👍

Reply:
I confirm that. It is not needed. So I changed to verify! Thanks 👍